### PR TITLE
Fabo/address component

### PR DESCRIPTION
**Pull request by faboweb**
To allow Polkadot addresses

### CODE REVIEW

**Comment by Bitcoinera** [comment]
I think this is deprecated right? `longForm` is not being used anymore

```
    longForm: {
      type: Boolean,
      required: false,
      default: false
    }
```
lines 30-34 in Address.vue (GitHub doesn't allow me to suggest any change there)

and lines 28-31 in Address.spec.js (the test for `longForm`)

**Comment by faboweb** [comment]
Good catch. Fixed.

File: tests/unit/specs/components/ActionModal/components/__snapshots__/DelegationModal.spec.js.snap [hunk 1]
```diff
@@ -10,7 +10,7 @@ exports[`DelegationModal Submission Data for Redelegating should set the subtota
       </span></div>
   </tmformgroup-stub>
   <tmformgroup-stub fieldid="to" fieldlabel="To" class="action-modal-form-group">
-    <tmfield-stub type="text" value="undefined - cosmosvaladdr…aladdr1kjisjsd862323" id="to" readonly=""></tmfield-stub>
+    <tmfield-stub type="text" value="undefined - cosmosvaladdr1kjisjs…aladdr1kjisjsd862323" id="to" readonly=""></tmfield-stub>
```
Comment (jbibla) [inline tests/unit/specs/components/ActionModal/components/__snapshots__/DelegationModal.spec.js.snap]: this doesn't look right

File: tests/unit/specs/components/common/__snapshots__/Address.spec.js.snap [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+exports[`Address Component should show a short address 1`] = `
+<div
+  class="bech32-address"
+>
+  <div
+    class="address"
+  >
+    
+    cosm…6789
```
Comment (jbibla) [inline tests/unit/specs/components/common/__snapshots__/Address.spec.js.snap]: this doesn't look right

File: tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap [hunk 3]
```diff
@@ -140,7 +132,7 @@ exports[`TmSessionExplore shows a form to sign in with an address 1`] = `
               class="tm-li-session-title"
             >
               <span>
-                cosmos…a7m64khj8xyz
+                cosmos1vxkye…a7m64khj8xyz
```
Comment (jbibla) [inline tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap]: doesn't look right

**Comment by jbibla** [comment]
bunch of snapshots don't look correct in this PR

**Comment by Bitcoinera** [comment]
> bunch of snapshots don't look correct in this PR

@faboweb what do you think?

**Comment by faboweb** [comment]
looks like it. will fix it. this looks like it has to do with the length parameter

**Comment by faboweb** [comment]
the addresses are not bech32 this is why they get formatted in a standard way.

**Pull request merged**